File: Distancia3PP/Punto.vb
Public Class Punto
    Protected x As Decimal

    Public Property X1() As Decimal
        Get
            Return x
        End Get
        Set(value As Decimal)
            x = value
        End Set
    End Property

    Public Sub New()
        x = 0
    End Sub

    Public Sub New(x As Decimal)
        Me.x = x
    End Sub

    Public Function Calcular_Distancia(Punto_a As Punto)
        Return Math.Abs(x - Punto_a.X1)
    End Function

    Public Overrides Function ToString() As String
        Return x + " "
    End Function

End Class

File: Distancia3P/Punto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Distancia3P
{
    class Punto
    {
        protected decimal x;

        public decimal X {
            get { return x; }
            set { x= value; } }

        public Punto(){
            x = 0;
          
}
        public Punto(decimal x)
        {
            this.X = x;
        }
          
        public decimal Calcular_Distancia(Punto a)
        {
            return Math.Abs(x - a.X);
            
        }

        public override string ToString()
        {
            return x + " ";
        }
    }
} 
